Build automation script

walteraa opened the pull request:
Now is possible to build all images which the Dockerfiles are inside the docker/ path.

Is important note that there is a pattern which the Dockerfile should respect: Dockerfile.$IMAGE_TAG.

walteraa committed:
Add an automation script to build images

Added a script to automate the build images using all Dockerfiles inside of docker/ path.
This Dockerfiles should respect the follow pattern: Dockerfile.$IMAGE_TAG.

walteraa committed:
Fix README

Fix the README text about Dockerfile pattern.

walteraa committed:
Add a script to test if an user has sudo access

This script verify if the user is sudo.
The script will be "deployed" in the container in the path /var/scripts/

The script receives a parameter($1) which is the user password.

walteraa committed:
Make the script faster

Now the docker build will run in paralel.

walteraa committed:
Add sudo installation and script deploy in dockerfiles

Added the command to run apt-get install sudo.
Added the command to deploy the "sudo test" script.

terciodemelo reviewed (dismissed):
We shouldn't have to deploy any kind of scripts to the hosts, even if they are test containers.

sshpm is intended to verify if the provided user for changes is sudo by other means rather then expecting a certain script to be present in the host system

walteraa committed:
Remove sudo test deploy

Remove the deployment of test_sudo.sh file from Dockerfiles and remove the file from workspace.
